admin/app/com/lucidchart/piezo/admin/controllers/HealthCheckController.scala

```diff
@@ -0,0 +1,27 @@
+package com.lucidchart.piezo.admin.controllers
+
+import org.joda.time.{DateTime,Minutes}
+import org.joda.time.format.ISODateTimeFormat
+import play.api._
+import play.api.Play.{current,configuration}
+import play.api.mvc._
+import scala.io.Source
+
+object HealthCheckController extends Controller {
+ def main = Action { implicit requests =>
+   if(workersAreHealthy) {
+     Ok
```

Comment:
How about we return the heartbeat timestamp in either case? I assume we want to go with json for the format of the response body.